Examples/LambdaFunctions/Sources/LambdaAPI/APIGateway+Extensions.swift

```diff
@@ -0,0 +1,73 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import AWSLambdaEvents
+
+public extension APIGateway.V2.Request {
+    func object<T: Codable>() throws -> T {
+        let decoder = JSONDecoder()
+        guard let body = self.body,
+            let dataBody = body.data(using: .utf8) else {
+                throw APIError.invalidRequest
+        }
+        return try decoder.decode(T.self, from: dataBody)
+    }
+}
+
+let defaultHeaders = ["Content-Type": "application/json",
+                      "Access-Control-Allow-Origin": "*",
+                      "Access-Control-Allow-Methods": "OPTIONS,GET,POST,PUT,DELETE",
+                      "Access-Control-Allow-Credentials": "true"]
+
+extension APIGateway.V2.Response {
+    
+    init(with error: Error, statusCode: AWSLambdaEvents.HTTPResponseStatus) {
+        
+        self.init(
+            statusCode: statusCode,
+            headers: defaultHeaders,
+            multiValueHeaders: nil,
+            body: "{\"message\":\"\(error.localizedDescription)\"}",
+            isBase64Encoded: false
+        )
+    }
+    
+    init<Out: Encodable>(with object: Out, statusCode: AWSLambdaEvents.HTTPResponseStatus) {
```

Comment:
see discussion in #129

Reply:
I made the decoder and encoder static. I suggest to keep it in this way until we decide what to do in #129.
7d39361